frontend/public/components/nav/items.tsx

```diff
@@ -83,6 +89,7 @@ class NavLink<P extends NavLinkProps> extends React.PureComponent<P> {
           onClick={onClick}
           title={tipText}
           data-tour-id={dataTourId}
+          data-cy="nav"
```

Comment:
This should be `data-test` per [our best practices](https://github.com/openshift/console/blob/master/frontend/packages/integration-tests-cypress/README.md#migrating-protractor-tests-to-cypress)